fe/src/main/java/org/apache/doris/load/loadv2/etl/EtlJobConfig.java

```diff
@@ -0,0 +1,548 @@
+                    '}';
+        }
+    }
+
+    public static class EtlFileGroup implements Serializable {
+        @SerializedName(value = "filePaths")
+        public List<String> filePaths;
+        @SerializedName(value = "fileFieldNames")
+        public List<String> fileFieldNames;
+        @SerializedName(value = "columnsFromPath")
+        public List<String> columnsFromPath;
+        @SerializedName(value = "columnSeparator")
+        public String columnSeparator;
+        @SerializedName(value = "lineDelimiter")
+        public String lineDelimiter;
+        @SerializedName(value = "isNegative")
+        public boolean isNegative;
+        @SerializedName(value = "fileFormat")
+        public String fileFormat;
+        @SerializedName(value = "columnMappings")
+        public Map<String, EtlColumnMapping> columnMappings;
+        @SerializedName(value = "where")
+        public String where;
+        @SerializedName(value = "partitions")
+        public List<Long> partitions;
+        @SerializedName(value = "hiveTableName")
+        public String hiveTableName;
+
+        public EtlFileGroup(List<String> filePaths, List<String> fileFieldNames, List<String> columnsFromPath,
+                            String columnSeparator, String lineDelimiter, boolean isNegative, String fileFormat,
+                            Map<String, EtlColumnMapping> columnMappings, String where, List<Long> partitions) {
+            this.filePaths = filePaths;
+            this.fileFieldNames = fileFieldNames;
+            this.columnsFromPath = columnsFromPath;
+            this.columnSeparator = columnSeparator;
+            this.lineDelimiter = lineDelimiter;
+            this.isNegative = isNegative;
+            this.fileFormat = fileFormat;
+            this.columnMappings = columnMappings;
+            this.where = where;
+            this.partitions = partitions;
+        }
+
+        @Override
+        public String toString() {
+            return "EtlFileGroup{" +
+                    "filePaths=" + filePaths +
+                    ", fileFieldNames=" + fileFieldNames +
+                    ", columnsFromPath=" + columnsFromPath +
+                    ", columnSeparator='" + columnSeparator + '\'' +
+                    ", lineDelimiter='" + lineDelimiter + '\'' +
+                    ", isNegative=" + isNegative +
+                    ", fileFormat='" + fileFormat + '\'' +
+                    ", columnMappings=" + columnMappings +
+                    ", where='" + where + '\'' +
+                    ", partitions=" + partitions +
+                    ", hiveTableName='" + hiveTableName + '\'' +
+                    '}';
+        }
+    }
+
+    public static class EtlColumnMapping implements Serializable {
+        @SerializedName(value = "functionName")
+        public String functionName;
+        @SerializedName(value = "args")
+        public List<String> args;
+        @SerializedName(value = "expr")
+        public String expr;
```

Comment:
better to add comment for this class, make it easy to understand the relation between functionName and expr